bodhi/client/__init__.py

```diff
@@ -787,5 +814,137 @@ def print_resp(resp, client, verbose=False):
             click.echo(caveat.description)
 
 
+@cli.group()
+def releases():
+    # Docs that show in the --help
+    """Interact with releases."""
+    # Developer Docs
+    """Manage the releases."""
+    pass  # pragma: no cover
+
+
+@releases.command(name='create')
+@handle_errors
+@add_options(release_options)
+@url_option
+def create_release(username, password, url, **kwargs):
+    """Create a release."""
+    client = bindings.BodhiClient(base_url=url, username=username, password=password)
+    kwargs['csrf_token'] = client.csrf()
+
+    save(client, **kwargs)
+
+
+@releases.command(name='edit')
+@handle_errors
+@add_options(release_options)
+@click.option('--new-name', help='New release name (eg: F20)')
+@url_option
+def edit_release(username, password, url, **kwargs):
+    """Edit an existing release."""
+    client = bindings.BodhiClient(base_url=url, username=username, password=password)
+    csrf = client.csrf()
+
+    edited = kwargs.pop('name')
+
+    if edited is None:
+        print("ERROR: Please specify the name of the release to edit")
+        return
+
+    res = client.send_request('releases/%s' % edited, verb='GET', auth=True)
+
+    data = munch.unmunchify(res)
+
+    if 'errors' in data:
+        print_errors(data)
+
+    data['edited'] = edited
+    data['csrf_token'] = csrf
+
+    new_name = kwargs.pop('new_name')
+
+    if new_name is not None:
+        data['name'] = new_name
+
+    for k, v in kwargs.items():
+        if v is not None:
+            data[k] = v
+
+    save(client, **data)
+
+
+@releases.command(name='info')
+@handle_errors
+@click.argument('name')
+@url_option
```

Comment:
Maybe we can add the `staging_option` here too and add it to the BodhiClient args.